test/java/seedu/addressbook/commands/ViewCommandTest.java

```diff
@@ -0,0 +1,161 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import static seedu.addressbook.util.TestUtil.assertCommandResult;
+
+public class ViewCommandTest {
+    private AddressBook typicalAddressBook;
+    private AddressBook emptyAddressBook;
+    private List<ReadOnlyPerson> emptyDisplayList = Collections.emptyList();
+    private List<ReadOnlyPerson> displayListWithAllTypicalPersons;
+    private List<ReadOnlyPerson> listWithSome;
+
+    @Before
+    public void setUp() {
+        emptyAddressBook = TestUtil.createAddressBook();
+
+        TypicalPersons td = new TypicalPersons();
+        typicalAddressBook = td.getTypicalAddressBook();
+        displayListWithAllTypicalPersons = Arrays.asList(td.getTypicalPersons());
+        listWithSome = Arrays.asList(td.amy, td.candy, td.dan);
```

Comment:
@pyokagan but it needs `td` the `TypicalPersons` instance to get the typical persons, if moved outside, needs to change all the relevant persons and methods to static?